src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+
+  actions.forEach((item) => {
+    const copyState = result.length
+      ? { ...result[result.length - 1] }
+      : { ...state };
```

Comment:
to get last element it's easier to use at(-1)

btw, not sure if you need it here, you can just create this variable outside of loop, work with it, and every time you push it -> just push it's copy

Reply:
This works on LTS node, but not on 14.21.3..... I don't understand why...